src/test/java/cms/gov/madie/measure/services/QicorePackageServiceTest.java

```diff
@@ -2,25 +2,81 @@
+import cms.gov.madie.measure.exceptions.InvalidResourceStateException;
+import cms.gov.madie.measure.factories.ModelValidatorFactory;
+import cms.gov.madie.measure.repositories.ExportRepository;
+import cms.gov.madie.measure.utils.MeasureUtil;
+import gov.cms.madie.models.measure.Export;
 import gov.cms.madie.models.measure.Measure;
+import gov.cms.madie.models.measure.MeasureMetaData;
+import gov.cms.madie.packaging.utils.PackagingUtilityFactory;
+import gov.cms.madie.packaging.utils.qicore411.PackagingUtilityImpl;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
-import org.mockito.Mock;
+import org.mockito.*;
 import org.mockito.junit.jupiter.MockitoExtension;
 
+import java.lang.reflect.InvocationTargetException;
+import java.util.Optional;
+
 import static org.hamcrest.CoreMatchers.*;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
+import static org.mockito.Mockito.doNothing;
 
 @ExtendWith(MockitoExtension.class)
 public class QicorePackageServiceTest {
   @Mock private BundleService bundleService;
+  @Mock MeasureService measureService;
+  @Mock FhirServicesClient fhirServicesClient;
+  @Mock private ExportRepository exportRepository;
+  @Mock private ModelValidatorFactory modelValidatorFactory;
+  @Mock private QiCoreModelValidator qicoreModelValidator;
+  @Mock private MeasureUtil measureUtil;
+
+  @Captor private ArgumentCaptor<Export> exportArgumentCaptor;
+
   @InjectMocks private QicorePackageService qicorePackageService;
 
+  private Measure existingMeasure;
+  private static final String TEST_USER = "test-user";
+  private static final String TEST_ACCESS_TOKEN = "test-access-token";
+  private static final String TEST_MEASURE_ID = "testMeasureId";
+  private static final String MODEL_QI_CORE = "QI-Core v4.1.1";
+  private static final String MEASURE_BUNDLE_JSON =
+      """
+              {"resourceType": "Bundle","entry": [ {
+                  "resource": {
+                    "resourceType": "Measure","text":{"div":"humanReadable"}}}]}""";
+  private static final String TEST_HUMAN_READABLE = "test human readable";
+  private static final String BUNDLE_OPERATION_ERROR =
+      "An error occurred while bundling Measure with ID testMeasureId. Please try again later or contact a System Administrator if this continues to occur.";
+
+  private static MockedStatic<PackagingUtilityFactory> factory;
+
+  @BeforeAll
+  public static void staticSetup() {
+    factory = Mockito.mockStatic(PackagingUtilityFactory.class);
```

Comment:
Codacy threw an error because `Mockito.*` is already astatic import. there are couple more such usages in this file.

Reply:
fixed